dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/common/FriendlyFileTypeUtil.java

```diff
@@ -0,0 +1,51 @@
+package edu.harvard.iq.dataverse.common;
+
+import edu.harvard.iq.dataverse.common.files.mime.ShapefileMimeType;
+import edu.harvard.iq.dataverse.persistence.datafile.DataFile;
+
+import java.util.MissingResourceException;
+
+public class FriendlyFileTypeUtil {
+
+    public static String getUserFriendlyFileType(DataFile dataFile) {
+        String fileType = dataFile.getContentType();
+
+        if (fileType != null) {
+            if (fileType.equalsIgnoreCase(ShapefileMimeType.SHAPEFILE_FILE_TYPE.getMimeValue())) {
+                return ShapefileMimeType.SHAPEFILE_FILE_TYPE.getFriendlyName();
+            }
+            if (fileType.contains(";")) {
+                fileType = fileType.substring(0, fileType.indexOf(";"));
+            }
+            try {
+                return BundleUtil.getStringFromPropertyFile(fileType, "MimeTypeDisplay");
+            } catch (MissingResourceException e) {
+                return fileType;
+            }
+        }
+
+        return fileType;
+    }
+    
+    
+    public static String getUserFriendlyOriginalType(DataFile dataFile) {
+        if (!dataFile.isTabularData()) {
+            return null;
```

Comment:
If you can return null it should be Optional method.

Reply:
I rather do not change this for now.
Method is used in datafile and there are couple of methods that concerns "original" file.
It will introduce inconsistency between rather connected between each other. DataFile methods. I think it should be done rather in one step

Reply:
If there is a risk of domino error's then I would also leave it for now. I wasn't sure if this was your class or not since I couldn't find the class with this name. And github says that it is a new class not renamed, so it is hard to keep track.

Reply:
Oh. It was originally part of FileUtil and I didn't want to move the whole class